portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import static com.google.sps.data.CommentsDatastore.deleteComments;
```

Comment:
nit: can you use CommentsDataStore.deleteElements in the doPost directly? the static import is probably not necessary here and we don't know where "deleteComments" is coming from.